### ISSUE TITLE
Rename default branch to main

### ISSUE THREAD

**Issue by Erisa** [Yuuki-Discord/Yuuki-Bot#76]
tldr if you just got here, run this on your local copy:
```
git pull origin main
git checkout main
git branch -D master
```
and all will be right in the world.

--
Now that GitHub has an easy way to do it: https://github.blog/changelog/2021-01-19-support-for-renaming-an-existing-branch/
I propose updating the default branch to be called `main`.

Aside from me having to update the actions workflows, this will require everyone with a local copy of the code to run some commands, which will be shown on the main page of the repository for every contributor:
![Screenshot_20210120-202631_Vivaldi](https://user-images.githubusercontent.com/14004943/105231391-b8c67780-5b5e-11eb-8141-e661335b5b92.png)

It's not a change to be made lightly on an active repository, maybe we can take advantage of this to also update the inconsistencies in author details of past commits? Suggestions welcomed.

**Comment by spotlightishere** [comment]
I am completely for both proposed actions! `main` is a good branch name and one I've taken to use myself where possible.

Similarly, the opportunity to rewrite author details would be much appreciated given earlier commits. I believe we discussed this at a point but never followed through due to the hassle of having everyone resync. Now that this action is simplified to pulling a separate branch with its own history anyway, there's no reason not to, in my opinion.

**Comment by Erisa** [comment]
Both changes should be ready to go now, changing default shortly after sending this.

Since we opted to not use githubs rename system, updating local copies should be a matter of
```
git pull origin main
git checkout main
git branch -D master
```

Keeping open for a little bit so people see.
